src/BlazorWebView/src/SharedSource/BlazorWebViewServiceCollectionExtensions.cs

```diff
@@ -44,17 +44,40 @@ public static IServiceCollection AddMauiBlazorWebView(this IServiceCollection se
 #elif WEBVIEW2_WPF
 			services.TryAddSingleton<WpfBlazorMarkerService>();
 #endif
+
+#if WEBVIEW2_MAUI
+			return new MauiBlazorWebViewBuilder();
+#elif WEBVIEW2_WINFORMS
+			return services;
+#elif WEBVIEW2_WPF
 			return services;
+#endif
 		}
 
 		/// <summary>
 		/// Enables Developer tools on the underlying WebView controls.
 		/// </summary>
 		/// <param name="services">The <see cref="IServiceCollection"/>.</param>
 		/// <returns>The <see cref="IServiceCollection"/>.</returns>
+#if WEBVIEW2_WINFORMS
 		public static IServiceCollection AddBlazorWebViewDeveloperTools(this IServiceCollection services)
```

Comment:
Did we explicitly decide whether to do this for WinForms/WPF too, or not? I don't remember, nor do I have a strong opinion at this time.

Reply:
They will do something similar, but there are dedicated issues for the WinForms and WPF cases, presumably to be handled in another PR: #5909, #5910